README.md

```diff
@@ -1,264 +1,505 @@
+# NAMETBD: A design system integrating to Pattern Lab and a Drupal 8 theme
+
 [![Build Status](https://travis-ci.org/phase2/pattern-lab-starter.svg?branch=master)](https://travis-ci.org/phase2/pattern-lab-starter)
 
-# Prerequisites
+NAMETBD is an opinionated set of tools and examples to:
+
+1. Build an application-agnostic **design system**
+1. Apply that design system to a locally-served **Pattern Lab** for rapid prototyping
+1. Apply that design system to a **Drupal theme**
+
+In depth documentation about frontend approach using this project at [Phase2 Frontend Docs](https://phase2.github.io/frontend-docs/)
+
+## Prerequisites
+
+- [Node `^8.0.0`](https://nodejs.org)
+- [PHP `5.6`, `^7.0.0`](https://php.net)
```

Comment:
Why support PHP 5.6? ("Because we support what Drupal 8 supports" is totally valid, but there are no major hosting providers of Drupal that impose this as a limitation now.)

Reply:
This came up in the DI room. It compiles in 5.6, just slower. So I guess we "support" it!